scripts/tools/categorylist.js

```diff
@@ -0,0 +1,3 @@
+const categoryList = ["generator", "code-first", "converters", "validators", "directories", "documentation generators", "dls", "frameworks", "ui components", "mocking and testing", "diff", "ci&cd", "editors"]
```

Comment:
@derberg Should we make the first letter of each category capital or exactly according to category heading we will show in UI? because then we don't have to manually change anything at frontend. We can directly use the key of the object to show the text in frontend.